types/react-router-redux/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "../tslint.json" }
+{
+    "extends": "../tslint.json",
+    "rules": {
+        "forbidden-types": false
```

Comment:
You shouldn't normally get lint errors for dependencies. I think the linting was in flux yesterday, which could be the cause of the redux lint errors.

In any case the name of the rule is `banned-types` so this `forbidden-types` shouldn't be doing anything anyway. I tested the branch and the set of lint errors is the same with and without the rule, so removing it is the right thing.